lib/constants.dart

```diff
@@ -0,0 +1,6 @@
+class Constants {
```

Comment:
I think class Constants is redundant.
You can write "const int defaultDate = 0000;"

Reply:
I planned in this project to use the Constants class for all possible constants
why is it redundant? Not good practice?

Reply:
sure, it is non-recommended practice.
[AVOID defining a class that contains only static members](https://dart.dev/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members)

Reply:
make import as constants